File: server/Models/Cult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using server.Models;

namespace Instacult_v2.Models
{
    public class Cult : RepoItem<int>
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string CoverImg { get; set; }
        public int MemberCount { get; set; }
        public int Fee { get; set; }
        public Boolean InvitationRequired { get; set; }
        public string LeaderId { get; set; }
        public Profile Leader { get; set; }
    }
}

File: client/src/models/Cult.js
export class Cult{
  constructor(data){
    this.id = data.id
    this.name = data.name
    this.description = data.description
    this.fee = data.fee
    this.coverImg = data.coverImg
    this.memberCount = data.memberCount
    this.invitationRequired = data.invitationRequired
    this.leaderId = data.leaderId
    this.leader = data.leader
  }
}
